SPECS/influxdb/influxdb.spec

```diff
@@ -18,7 +18,7 @@
 Summary:        Scalable datastore for metrics, events, and real-time analytics
 Name:           influxdb
 Version:        2.6.1
-Release:        25%{?dist}
+Release:        24%{?dist}
```

Comment:
It looks like this revision has already been published: https://packages.microsoft.com/cbl-mariner/2.0/prod/base/x86_64/Packages/i/influxdb-2.6.1-24.cm2.x86_64.rpm

Shall we increase the release number instead?